ERCS/erc-7683.md

```diff
@@ -65,28 +82,24 @@ A compliant cross-chain order type MUST be convertible into the `ResolvedCrossCh
 /// @dev Defines all requirements for filling an order by unbundling the implementation-specific orderData.
 /// @dev Intended to improve integration generalization by allowing fillers to compute the exact input and output information of any order
 struct ResolvedCrossChainOrder {
-	/// @dev The contract address that the order is meant to be settled by.
-	address settlementContract;
-	/// @dev The address of the user who is initiating the swap
-	address swapper;
-	/// @dev Nonce to be used as replay protection for the order
-	uint256 nonce;
+	/// @dev The address of the user who is initiating the transfer
+	address user;
 	/// @dev The chainId of the origin chain
 	uint32 originChainId;
 	/// @dev The timestamp by which the order must be initiated
 	uint32 initiateDeadline;
 	/// @dev The timestamp by which the order must be filled on the destination chain(s)
 	uint32 fillDeadline;
 
-	/// @dev The inputs to be taken from the swapper as part of order initiation
-	Input[] swapperInputs;
-	/// @dev The outputs to be given to the swapper as part of order fulfillment
-	Output[] swapperOutputs;
+	/// @dev The inputs to be taken from the user as part of order initiation
+	Input[] userInputs;
+	/// @dev The outputs to be given to the user as part of order fulfillment
+	Output[] maxOutputs;
```

Comment:
+1 for max and min outputs

Reply:
Yeah, I realize there's a typo here. maxOutputs should be maxUserOutputs I think.